python/paddle_fl/mobile/trainer/language_model_trainer.py

```diff
@@ -270,10 +297,13 @@ def generate_program_desc(self, do_test=False):
         """
         with fluid.program_guard(self.main_program_, self.startup_program_):
             self.input_model_ = LanguageModel()
-            model_configs = {}
+            model_configs = self.trainer_config
             self.input_model_.build_model(model_configs)
+
             optimizer = fluid.optimizer.SGD(
-                learning_rate=self.trainer_config["lr"])
+                learning_rate=self.trainer_config["lr"],
+                grad_clip=fluid.clip.GradientClipByGlobalNorm(
```

Comment:
The `grad_clip` parameter is added to SGDOptimizer since 1.8.0, we should add a requirements(PaddlePaddle >= 1.8.0) to README.txt.

Reply:
README has been updated